mockgcp/apis/mockgcp/cloud/ids/v1/service.proto

```diff
@@ -13,187 +13,285 @@
 // limitations under the License.
 
 syntax = "proto2";
-package google.cloud.ids.v1;
+package mockgcp.cloud.ids.v1;
```

Comment:
Do you know what happened here?

Reply:
I think this proto was originally generated before https://github.com/GoogleCloudPlatform/k8s-config-connector/commit/bfe519eddf03983d049084ccd32abb072d552477, which retroactively changed the package, but did not re-generate the protos.

Reply:
Actually just realized -- maybe this means that this PR didn't really fix the comments, it was just the first time in a while that we've re-generated the protos. Hopefully this doesn't cause issues for the services with regenerated protos.

Reply:
I was wondering the same, but all the presubmit test passes. I spot checked some proto code diff, seems to be all backward compatible (which makes sense because KCC only add GA-ed resources)